### PR TITLE
docs: clarifies the service limitation

### PR DESCRIPTION
**Pull request by sonukapoor**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #36332

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by ajitsinghkaler** [comment]
@sonukapoor commit-msg is not correct please add fixes #36332 so that that issue closes automatically once this pull request closes

**Comment by sonukapoor** [comment]
@pkozlowski-opensource Can you please review this?

File: aio/content/guide/providers.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ Though you can provide services by lazy loading modules, not all services can be
 Another way to limit provider scope is by adding the service you want to limit to the component’s
 `providers` array. Component providers and NgModule providers are independent of each other. This
 method is helpful when you want to eagerly load a module that needs a service all to itself.
-Providing a service in the component limits the service only to that component (other components in
-the same module can’t access it).
+Providing a service in the component limits the service only to that component and its descendants
+(other components in the same module can’t access it).
```
Comment (aikidave) [inline aio/content/guide/providers.md:62]: Readers tend to skip content in parentheses. I recommend avoiding them. 

```suggestion
Other components in the same module can’t access it.
```

File: aio/content/guide/providers.md [hunk 2]
```diff
@@ -58,8 +58,8 @@ Though you can provide services by lazy loading modules, not all services can be
 Another way to limit provider scope is by adding the service you want to limit to the component’s
 `providers` array. Component providers and NgModule providers are independent of each other. This
 method is helpful when you want to eagerly load a module that needs a service all to itself.
-Providing a service in the component limits the service only to that component (other components in
-the same module can’t access it).
+Providing a service in the component limits the service only to that component and its descendants
```
Comment (aikidave) [inline aio/content/guide/providers.md:61]:
```suggestion
Providing a service in the component limits the service only to that component and its descendants.
```
(This goes along with my other suggestion below.

**Review by aikithoughts — CHANGES_REQUESTED** [review]

File: aio/content/guide/providers.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ Another way to limit provider scope is by adding the service you want to limit t
 `providers` array. Component providers and NgModule providers are independent of each other. This
 method is helpful when you want to eagerly load a module that needs a service all to itself.
 Providing a service in the component limits the service only to that component and its descendants
```
Comment (aikidave) [inline aio/content/guide/providers.md]: I think you still need a period at the end of line 61.

**Review by aikithoughts — CHANGES_REQUESTED** [review]

File: aio/content/guide/providers.md (hunk 3 above)
Comment (sonukapoor) [reply aio/content/guide/providers.md]: Yes, I was trying the GitHub suggestion feature. It didn't work properly. I will download the branch and make the update today. Thanks for reviewing it.

Comment (sonukapoor) [reply aio/content/guide/providers.md]: This should be good now. Please re-review.

**Review by aikithoughts — APPROVED** [review]

**Comment by sonukapoor** [comment]
@aikidave / @pkozlowski-opensource Any chance to get this merged?

**Review by kara — APPROVED** [review]
LGTM

**Pull request closed without merging**